index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400|Roboto&display=swap" rel="stylesheet">  
+    <title>Apple- clone</title>
+</head>
+<body>
+    <div class="top container">
+        <header>
+            <nav>
+                <ul>
+                    <li><a href="#"> <img src="assets/Apple-Logo.png" alt="Apple"> </a></li>
+                    <span></span>
+                    <li><a href="#">Store</a></li>
+                    <span></span>
+                    <li><a href="#">Mac</a></li>
+                    <span></span>
+                    <li><a href="#">iPod</a></li>
+                    <span></span>
+                    <li><a href="#">iPhone</a></li>
+                    <span></span>
+                    <li><a href="#">iPad</a></li>
+                    <span></span>
+                    <li><a href="#">iTunes</a></li>
+                    <span></span>
+                    <li><a href="#">Support</a></li>
+                    <span></span>
+                    <li><input type="text"></li>
+                </ul>
+            </nav>
+            <a href="#"><h1>What will your verce be?</h1> <br>
```

Comment:
You need to match the same approach of the original page to get approved in this project, meaning that you have to use images as a background to display these texts. There 3 headings in the original page and a background image to each one. You must have an anchor tag for each one (3 anchor tag) and inside the anchor tag a div. Use the div's background to display the images with text.